pkg/api/graph/interfaces.go

```diff
@@ -107,5 +105,5 @@ func (m ByKey) Less(i, j int) bool {
 type Suggestion string
 
 func (s Suggestion) String() string {
-	return fmt.Sprintf("try: %s", string(s))
```

Comment:
I am not arguing that this info is not useful in general. I am arguing that it lays in the wrong place. The reason we added `Suggestion` in the markers was to suggest commands that users could directly run and resolve their issues or at least dig further.